build/git-history.js

```diff
@@ -48,7 +48,9 @@ function getFromGit(contentRoot = CONTENT_ROOT) {
       hash = data[0];
       date = new Date(data[1]);
     } else if (line) {
-      const relPath = path.relative(realContentRoot, path.join(repoRoot, line));
```

Comment:
Maybe look at https://nodejs.org/api/path.html#path_path_posix if you always want the POSIX path style